src/CommonMarkConverter.php

```diff
@@ -58,4 +58,20 @@ public function convertToHtml($commonMark)
 
         return $this->htmlRenderer->renderBlock($documentAST);
     }
+
+    /**
+     * @return DocParser
+     */
+    public function getDocParser()
+    {
+        return $this->docParser;
+    }
+
+    /**
+     * @return HtmlRenderer
+     */
+    public function getHtmlRenderer()
+    {
+        return $this->htmlRenderer;
+    }
```

Comment:
> when all I want is to change one thing in the htmlRenderer

There's nothing you can really change though - if there was, then I'd be okay with this change.